common/Makefile.common.mk

```diff
@@ -71,9 +71,12 @@ code-gen:
 	# Workaround for relative/absolute path issue
 	# see https://github.com/IBM/meta-operator/pull/32
 	@echo Updating the CRD files with the OpenAPI validations
-	GOPATH=/tmp operator-sdk generate openapi
+	# Build the latest openapi-gen from source
+	which ./bin/openapi-gen > /dev/null || go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
```

Comment:
can we put it under a hidden directory? such as `build/.bin`.

Reply:
I will put the `openapi-gen` into `build/_generate` and add `build/_generate` into the .gitignore